sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Common/Exceptions/AuthenticationEventTriggerRequestValidationException.cs

```diff
@@ -12,7 +12,10 @@ namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
     internal class AuthenticationEventTriggerRequestValidationException : AuthenticationEventTriggerValidationException
     {
         /// <inheritdoc/>
-        public override HttpStatusCode ExceptionStatusCode => HttpStatusCode.InternalServerError;
+        public override HttpStatusCode ExceptionStatusCode => HttpStatusCode.BadRequest;
+
+        /// <inheritdoc/>
```

Comment:
For later, we should see if there is a common library that we can use to pull these strings given the exception status code.

Reply:
NOT FOR THIS PR. Just a note for us.